### DISCUSSION TITLE
Invalid tokens being returned

### DISCUSSION THREAD

**Discussion by briannielson** [etsy/open-api#1172] [Q&A]
Hello,

Starting at around 12pm Pacific today, we started receiving invalid tokens from the `https://api.etsy.com/v3/public/oauth/token` endpoint. When we attempt to use these tokens for other API requests, we get a 401 error. We are not receiving errors from the oauth/token endpoint.

This is happening for about 22/120 of our customers. Were there any changes on your end that could explain this? Otherwise we need to keep digging more... LMK if you need an example.

**Comment by Gareth064** [comment]
I haven't experienced this myself recently, but last time I did it was because the customers had changed their Etsy Password and I think that means they have to re-authenticate your app again.
